.github/workflows/go.yml

```diff
@@ -7,30 +7,35 @@ on:
     branches: [ main ]
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
+
     steps:
     - uses: actions/checkout@v2
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.15
+        go-version: 1.16.3
+
     - name: Build
       run: go build -v
+
     - name: Test
       run: go test -v ./...
+
     - name: golangci-lint
       uses: golangci/golangci-lint-action@v2.5.1
       with:
-        version: v1.37.1
+        version: v1.39.0
```

Comment:
Perhaps not specific to this PR, but on a related note..It would be really good if we had some sort of dependabot type character who can make sure all these actions are update and we have version parity over all the repos for go version, linters etc. Interested to hear if you have any ideas @cpanato

Reply:
sounds interesting, but I don't think to exist this tool yet, but we can build such action to help on that :) I will think about how to do that